File: java/claude/p10_fc.java
import java.util.Scanner;

public class p10_fc {

    private static final int[] ID_SEQUENCE = {10, 20, 30, 40, 50};

    /**
     * Returns the value at {@code array[index]} if the index is valid,
     * or -1 if index is out of bounds.
     */
    public static int getValueFromArray(int[] array, int size, int index) {
        if (array == null || size <= 0 || index < 0 || index >= size) {
            return -1;
        }
        return array[index];
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        String input = scanner.nextLine().trim();

        int index;
        try {
            index = Integer.parseInt(input);
        } catch (NumberFormatException e) {
            System.exit(-1);
            return;
        }

        int result = getValueFromArray(ID_SEQUENCE, ID_SEQUENCE.length, index);
        System.exit(result);
    }
}

File: python/claude/p10_fc.py
import sys

ID_SEQUENCE = [10, 20, 30, 40, 50]


def get_value_from_array(array, size, index):
    """Returns array[index] if index is valid (0 <= index < size), else -1."""
    if array is None or size <= 0 or index < 0 or index >= size:
        return -1
    return array[index]


def main():
    raw = sys.stdin.readline().strip()

    try:
        index = int(raw)
    except ValueError:
        sys.exit(-1)

    result = get_value_from_array(ID_SEQUENCE, len(ID_SEQUENCE), index)
    sys.exit(result)


if __name__ == "__main__":
    main()
